tests/src/test/java/alluxio/client/rest/S3ClientRestApiTest.java

```diff
@@ -821,6 +823,43 @@ public void completeMultipartUpload() throws Exception {
     }
   }
 
+  @Test
+  public void bucketTags() throws Exception {
+    createBucketRestCall("test0");
+    createBucketRestCall("test1");
+
+    assertThat(getObjectTagRestCall("test0"), new StringContains("<TagSet/>"));
+    assertThat(getObjectTagRestCall("test1"), new StringContains("<TagSet/>"));
+
+    final HashMap<String, String> tag0 = new HashMap<>();
+    tag0.put("abc", "def");
+    tag0.put("ghi", "jkl");
+    putObjectTagRestCall("test0", createTagXml(tag0));
+
+    final String tag0result = getObjectTagRestCall("test0");
+    assertThat(tag0result, new StringContains("<Key>abc</Key><Value>def</Value>"));
+    assertThat(tag0result, new StringContains("<Key>ghi</Key><Value>jkl</Value>"));
+    assertThat(getObjectTagRestCall("test1"), new StringContains("<TagSet/>"));
```

Comment:
Is this string NOT present when tags are returned?

Reply:
Yeah in particular, <TagSet/> means that TagSet tag is immediately closed. Which wouldn't happen if there are any members inside a TagSet.